plugins/osdn/api/types.go

```diff
@@ -1,13 +1,10 @@
 package api
 
 import (
-	knetwork "k8s.io/kubernetes/pkg/kubelet/network"
 	pconfig "k8s.io/kubernetes/pkg/proxy/config"
 )
 
 type OsdnNodePlugin interface {
-	knetwork.NetworkPlugin
-
 	Start() error
 }
```

Comment:
It's more complicated for the node, because the OSDN plugin gets constructed (node_config.go::BuildKubernetesNodeConfig()) in a different place than it gets started (node.go::RunPlugin()).